etc/picongpu/jureca-jsc/gpus_picongpu.profile.example

```diff
@@ -0,0 +1,112 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj=$(groups | awk '{print $5}')
+
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="nano"
+
+# General modules #############################################################
+#
+module purge
+module load GCC/7.3.0
+module load CUDA/9.2.88
+module load CMake/3.13.0
+module load MVAPICH2/2.3-GDR
```

Comment:
is this MPI build with CUDA awareness? we have to make sure, otherwise we have to control MPI pinning to avoid steels between network and cuda pinned pages.

Reply:
The docs and how the module is exposed say it is CUDA-aware. I did not do any specific checks.